numba_cuda/numba/cuda/simulator/api.py

```diff
@@ -91,6 +92,9 @@ def jit(func_or_sig=None, device=False, debug=False, argtypes=None,
     if link is not None:
         raise NotImplementedError('Cannot link PTX in the simulator')
 
+    debug = config.CUDA_DEBUGINFO_DEFAULT if debug is None else debug
+    opt = (config.OPT._raw_value != 0) if opt is None else opt
```

Comment:
`opt` isn't used so I don't think the logic on this line needs to be added here.

I've run flake8 locally and I'm not sure why it doesn't pick this up - maybe because it's a parameter?

Reply:
removed `opt` line.